Makefile

```diff
@@ -4,13 +4,13 @@ css ?= static/css/main.css
 all: styles
 
 staging:
-	cd deploy/ && fab deploy && cd ..
+	cd deploy/ && fab deploy --set environment=staging && cd ..
 
 production:
 	cd deploy/ && fab deploy --set environment=production && cd ..
 
-run:
-	python pulse.py
+debug:
+	DEBUG=true python pulse.py
```

Comment:
If the app is run with `DEBUG=true`, it will use Flask's debug mode (and show stack traces, etc.).